Python 3 compatibility.

JNRowe opened the pull request:
Love `expecter`, so shockingly simple and concise!

---

Included is a very minor fix for use with Python 3.  This breaks Python < 2.5 compat, but the tests didn't pass before hand so perhaps that isn't an issue?

Thanks,

James

JNRowe committed:
Python 3 compatibility.

The exception handling syntax means 2.5 support has been dropped, but the tests
didn't work on 2.5 before so perhaps it doesn't matter.

garybernhardt commented:
Thanks!

garybernhardt commented:
I added a tox.ini that runs the tests against 2.6, 2.7, 3.1, and 3.2 in d3b8063.

JNRowe commented:
Thanks for merging!

> I added a tox.ini that runs the tests against 2.6, 2.7, 3.1, and 3.2

Not sure if you'll be interested... but I added a [travis-ci](http://travis-ci.org/) config in  [feat/testing](https://github.com/JNRowe/expecter/tree/feat/testing).  Hadn't planned on submitting it to you, as I was just playing around with it the other night.  You can see the results at http://travis-ci.org/#!/JNRowe/expecter .

_Update_: And, now I see there is a new pull request with travis support in #7.  Should have looked before commenting ;)
